website/docs/d/auditing.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: network_peering"
+sidebar_current: "docs-mongodbatlas-datasource-network-peering"
+description: |-
+    Describes a Network Peering.
+---
+
+# mongodbatlas_network_peering
```

Comment:
Looks like this was a bad copy-pasta? This is describing peering instead of auditing.

Reply:
I renamed the and removed the right file :S